setup.py

```diff
@@ -58,7 +58,8 @@
     ],
     extras_require={
         'S3': ["awscli>=1.16.291"],
-        'GCS': ["google-cloud-storage>=1.7.0"]
+        'GCS': ["google-cloud-storage>=1.7.0"],
+        'GRPC': ["protobuf>=3.12.0", "grpcio>=1.29.0", "grpcio-health-checking-1.29.0", "grpcio-tools>=1.29.0"]
```

Comment:
Since the code is there and it is imported, won't the grpc requirements be needed in all scenarios?

Reply:
Indeed. Moved them to install_requires